### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by qin-guan** [sst-inc/help#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with members of SST Inc. We hope that you:
  * Ask questions you’re wondering about.
  * Help others as much as possible.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself.

**Comment by haikeyidesu** [comment]
Are we still meeting physically for SST inc?

**Reply by iso4096** [reply]
I doubt so.

**Reply by qin-guan** [reply]
Nope, not for now

**Reply by haikeyidesu** [reply]
Thanks

**Comment by senkyte** [comment]
If we are doing HBL today, where is the meeting or zoom link? (For React Native)

**Reply by qin-guan** [reply]
For RN, there won't be a meeting you need to join today, it will be own time own target. More details will be release on Google Classroom

**Comment by ultimate-supreme-leader** [comment]
heyyy, i don't understand how to use xcode so is there anyone who can basically tutor me? i understand if everyone's busy. in that case, can i just have some recommended resources to help me learn? thank youu!

**Reply by DakshRocks21** [reply]
What exactly you don't understand?

**Comment by haikeyidesu** [comment]
I'm not able to join the zoom as there are already 100 participants. What do I do?

**Reply by haikeyidesu** [reply]
ok I managed to join. Thanks
